plugins/obs-qsv11/obs-qsv11.c

```diff
@@ -979,6 +1002,9 @@ static bool obs_qsv_encode_tex(void *data, uint32_t handle, int64_t pts,
 {
 	struct obs_qsv *obsqsv = data;
 
+	// Added for debug of adding direct ARGB encoding
+	blog(LOG_INFO, "=== [qsv] receive texture, handle %p", handle);
+
```

Comment:
Change to LOG_DEBUG, or use an ifdef, or remove this.

If keeping this, please note that this spams the log on every frame for any GPU encoder, such as NVENC, with messages about QSV.